### PR TITLE
Reduce memory leak for tests

### PR DESCRIPTION
**Pull request by B-Galati**

### CODE REVIEW

File: src/DataFixtures/ORM/LoadMissionTypeData.php [hunk 1]
```diff
@@ -22,6 +23,12 @@ class LoadMissionTypeData implements FixtureInterface
 
     public function load(ObjectManager $manager)
     {
+        if (!$manager instanceof  EntityManagerInterface) {
```
Comment (salahm) [inline src/DataFixtures/ORM/LoadMissionTypeData.php]: you may use a trait here, with a function which takes 2 args : $manger, $tableName

File: app/config/config_test.yml [hunk 2]
```diff
@@ -111,3 +111,10 @@ old_sound_rabbit_mq:
 
 lexik_paybox:
     public_key: "%env(SSL_PUBLIC_KEY)%"
+
+doctrine:
+    dbal:
+        connections:
```
Comment (salahm) [inline app/config/config_test.yml]: you may also consider using at least an arrayCache pour doctrine metadata ;)

File: src/DataFixtures/ORM/LoadMissionTypeData.php (hunk 1 above)
Comment (Remg) [reply src/DataFixtures/ORM/LoadMissionTypeData.php]: `$manger` & `$tableName` are great var names, same lexical field and all... :trollface:

File: app/config/config_test.yml (hunk 2 above)
Comment (B-Galati) [reply app/config/config_test.yml]: We will do that with [DoctrineTestBundle](https://github.com/dmaicher/doctrine-test-bundle) in another PR

File: src/DataFixtures/ORM/LoadMissionTypeData.php (hunk 1 above)
Comment (B-Galati) [reply src/DataFixtures/ORM/LoadMissionTypeData.php]: I do agree

Comment (B-Galati) [reply src/DataFixtures/ORM/LoadMissionTypeData.php]: with @salahm lol

**Comment by B-Galati** [comment]
Closing, I'll do another PR will all changes.

**Pull request closed without merging**